CONTRIBUTING.rst

```diff
@@ -36,5 +37,128 @@ Pull Request. Contributions may include:
 * Helping to onboard new contributors
 * Other related activities
 
+Development
+===========
 
-Check `README.rst <README.rst>`_ development section instructions.
\ No newline at end of file
+Requirements
+-------------
+
+- Python >=3.9
+- Pipenv
+
+Getting the source code
+-----------------------
+
+`Fork <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_ the
+repository on `GitHub <https://github.com/vmware/repository-service-tuf-cli>`_ and
+`clone <https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository>`_
+it to your local machine:
+
+.. code-block:: console
+
+    git clone git@github.com:YOUR-USERNAME/repository-service-tuf-cli.git
+
+Add a `remote
+<https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork>`_ and
+regularly `sync <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork>`_ to make sure
+you stay up-to-date with our repository:
+
+.. code-block:: console
+
+    git remote add upstream https://github.com/vmware/repository-service-tuf-cli
+    git checkout main
+    git fetch upstream
+    git merge upstream/main
+
+Preparing the environment
+-------------------------
+
+After installing Python, install the pipenv tool:
+
+.. code:: shell
+
+    $ pip install pipenv
+
+
+Create a virtual environment for this project:
+
+.. code:: shell
+
+    $ pipenv shell
+
+
+Install the requirements from the Pipfile.
+
+The flag -d will install the development requirements:
+
+.. code:: shell
+
+    $ pipenv install -d
+
+
+.. note::
```

Comment:
I am not familiar with the `.rst` file format, but is there a header option that is below the one we use at line 74 for `Preparing the environment` and could be use to underline this note?

I would also underline lines 112 and 127 as it doesn't visually highlight those points:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/16246778/203800792-85d0883d-c22c-4b8f-ae9a-3420d19b2f79.png">

Reply:
> I am not familiar with the .rst file format, but is there a header option that is below the one we use at line 74 for Preparing the environment and could be use to underlin

A proper rst rendering would show the note appropriately, but GH does not support this (see https://github.com/github/markup/issues/68, https://github.com/github/markup/issues/72)

> I would also underline lines 112 and 127 as it doesn't visually highlight those points:

Good point, I will use a subsection (I need to try it out to see if it's rendered correctly by GH, so I will push a new commit)

Reply:
`note:` at line 99 is still not underlined.

Reply:
> `note:` at line 99 is still not underlined.

For the other parts I added the appropriate rst headings since they were actually subsections denoted with a semicolumn (":").
As far as the note goes, as I mentioned, it is rst valid but GH does not support rendering it. If our goal is everything to display to GH properly then maybe we should consider changing the file to markdown. 
Of course we should consider if there was another reason we chose rst for the README (@kairoaraujo maybe you know a specific reasoning behind this choice?). One thing I can think of is PyPI, but as far as I know it also supports [markdown](https://packaging.python.org/en/latest/guides/making-a-pypi-friendly-readme/#creating-a-readme-file). Maybe because there was a probability to re-use this file in Sphinx documentation?

Reply:
The choice of RST is for the Read The Docs Sphinx and some complex features, such as `include` with cutting parts, to avoid managing the exact text in multiple files.
In the umbrella repository, we compile all documentation together.

``note:`` is not supported by GH, but it is ok. Also, the Markdown doesn't have a proper 'note'.